packages/cacti-copm-core/src/main/proto/generated/openapi/models/view_address_v1_pb.proto

```diff
@@ -0,0 +1,26 @@
+/*
+  Hyperledger Cacti Plugin - Common Operator Primitives
+
+  Contains/describes the Hyperledger Cacti Common Operator Primitives plugin.
+
+  The version of the OpenAPI document: 2.0.0-rc.2
+
+  Generated by OpenAPI Generator: https://openapi-generator.tech
+*/
+
+syntax = "proto3";
+
+package org.hyperledger.cacti.plugin.cacti.plugin.copm.core;
+
+import "google/protobuf/any.proto";
+
+
+message ViewAddressV1PB {
+
+  string contract_id = 512823451;
```

Comment:
We may need more fields to this, but I'm assuming, we will keep updating this in future with implementations of these services?

Reply:
Yes.  For now I want to see what we can get away with.  In Corda, flowpath (for instance) is very similar to a contract specification.